### DISCUSSION TITLE
How to connect to the database?

### DISCUSSION THREAD

**Discussion by martijn-moveyou** [thecodingmachine/graphqlite#699] [Q&A]
I have a symfony application that is working. I add the thecodingmachine graphql dependency using composer and implement the demo graphql (The one with the demo query ```{ hello(name: "1234") }```

I do a post to my application running on port 8000 using postman. I do a post call to https://localhost:8000/graphql and that works great. I get the expected result.  Now want to retrieve information from the database. I have tried to inject the entitymanager of symfony in the constructor and no matter what I do get lots of errors. How do I retrieve information from the database?

**Comment by oojacoboo** [comment]
@martijn-moveyou I'm afraid that question is a bit too broad for this repository.  There are a LOT of basics involved here.  Firstly, you need to learn how to read stack traces.  "lots of errors" tells me you don't know how to debug issues.  Start there.

Also, this issue is likely not related to GraphQLite.
